classes/Product.php

```diff
@@ -6429,7 +6433,7 @@ public function hasAllRequiredCustomizableFields(Context $context = null)
      */
     public static function getOldTempProducts()
     {
-        $sql = 'SELECT id_product FROM `' . _DB_PREFIX_ . 'product` WHERE state=' . \Product::STATE_TEMP . ' AND date_upd < NOW() - INTERVAL 1 DAY';
+        $sql = 'SELECT id_product FROM `' . _DB_PREFIX_ . 'product` WHERE state=' . Product::STATE_TEMP . ' AND date_upd < NOW() - INTERVAL 1 DAY';
```

Comment:
```suggestion
        $sql = 'SELECT id_product FROM `' . _DB_PREFIX_ . 'product` WHERE state=' . static::STATE_TEMP . ' AND date_upd < NOW() - INTERVAL 1 DAY';
```

Suggestion

Reply:
I think this was changed by cs fixer or something, I didin't manually change this, so I'll leave it as is :smile:

Reply:
Yeah no problem, it was just a suggestion anyway 🙂